include/z64bgcheck.h

```diff
@@ -52,10 +52,51 @@ typedef struct {
 } CollisionPoly; // size = 0x10
 
 typedef struct {
-    /* 0x00 */ u16 cameraSType;
-    /* 0x02 */ s16 numCameras;
-    /* 0x04 */ Vec3s* camPosData;
-} CamData;
+    /* 0x00 */ Vec3s pos;
+    /* 0x06 */ Vec3s rot;
+    /* 0x0C */ s16 fov;
+    /* 0x0E */ union {
+                s16 jfifId;
+                s16 timer;
+                s16 flags;
+    };
+    /* 0x10 */ s16 unk_10; // unused
+} SubBgCamData;
+
+/**
+ * BgCamData Summary:
+ * 
+ * setting - camera setting described by CameraSettingType enum
+ * numData - The total count of Vec3s data in the collision
+ * data - data stored in Vec3s with various purposes summarized below:
+ * 
+ * numData = 0:
+ *      data unused
+ * 
+ * numData = 3: data organized in SubBgCamData struct
+ *      data[0]   // Position
+ *      data[1]   // Rotation
+ *      data[2].x // Field of View
+ *      data[2].y // Jfif Id, timer, flags
+ *      data[2].z // unused
```

Comment:
The array access stuff seems redundant if the data is supposed to be accessed with the struct anyway, so I would just omit it and keep only the mention of the struct

Reply:
I suppose it was here just to contract with the way data is stored with crawlspaces, but because crawlspaces is such a unique exception to how this data is used, I can just remove this